src/cfg.rs

```diff
@@ -67,13 +71,15 @@ impl Default for KatexConfig {
             macros: None,
             block_delimiter: Delimiter::same("$$".into()),
             inline_delimiter: Delimiter::same("$".into()),
+            pre_render: cfg!(feature = "pre-render"),
         }
     }
 }
 
 impl KatexConfig {
     /// Configured output type.
     /// Defaults to `Html`, can also be `Mathml` or `HtmlAndMathml`.
+    #[cfg(feature = "pre-render")]
```

Comment:
Suggestion: group all the `pre-render` functionalities into separate `mod`s, so we don't have `#[cfg(…)]` everywhere.